2-mandatory/3-financial-times.js

```diff
@@ -4,34 +4,76 @@
     The home page of the web site has a headline section, which only has space for article titles which are 65 characters or less.
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
+
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    return allArticleTitles.filter(title => title.length <= 65)
+
+    //  let newArray = [];
+    // for (let i = 0; i < allArticleTitles.length; i++){
+    //     if (allArticleTitles[i].length <= 65) {
+    //         newArray.push(allArticleTitles[i])
+    //     }
+    // }
+    // return newArray;
 }
+ 
 
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
     (you can assume words will always be seperated by a space)
 */
+
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+     let array = allArticleTitles[0].split(" ").length;
+    let spaceCount;
+    for (let i = 0; i < allArticleTitles.length; i++) {
+        wordCount = allArticleTitles[i].split(" ").length;
+        if (wordCount < array) {
+            array = wordCount;
+            spaceCount = i;
+        }
+    }
+    return allArticleTitles[spaceCount];
 }
 
+
 /*
     The editor of the FT has realised that headlines which have numbers in them get more clicks!
     Implement the function below to return a new array containing all the headlines which contain a number.
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+    return allArticleTitles.filter(headline => /[0-9]/.test(headline));
```

Comment:
I like this solution

Reply:
Why do you like it? What is appealing about this code?

Reply:
My comment is a sample of how beginners comment who make a review for beginners)) 
I liked that it fits in one line and it is clear what is happening. Although I trust for loop more.